apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Snippets/UserGuideSnippets.cs

```diff
@@ -355,12 +355,19 @@ public class City
 
             FirestoreChangeListener listener = query.Listen(snapshot =>
             {
-                Console.WriteLine($"Callback received snapshot");
+                Console.WriteLine($"Callback received query snapshot");
                 Console.WriteLine($"Count: {snapshot.Count}");
                 Console.WriteLine("Changes:");
                 foreach (DocumentChange change in snapshot.Changes)
                 {
-                    Console.WriteLine($"{change.Document.Reference.Id}: ChangeType={change.ChangeType}; OldIndex={change.OldIndex}; NewIndex={change.NewIndex})");
+                    var document = change.Document;
+                    Console.WriteLine($"{document.Reference.Id}: ChangeType={change.ChangeType}; OldIndex={change.OldIndex}; NewIndex={change.NewIndex})");
+                    if (document.Exists)
+                    {
+                        var name = document.GetValue<string>("Name");
```

Comment:
And here.
I know it's very obvious what the types are here, but still think we should not use `var` anywhere in documentation.